docs/docsite/rst/installation_guide/installation_distros.rst

```diff
@@ -7,31 +7,35 @@ Installing Ansible on specific operating systems
 
 The ``ansible`` package can always be :ref:`installed from PyPI using pip <intro_installation_guide>` on most systems but it is also packaged and maintained by the community for a variety of Linux distributions.
 
-The following instructions will guide you through installing the ``ansible`` package with your preferred distribution's package manager.
+This document guides you through installing the ``ansible`` package with different package managers.
 
-.. note:: For maintainers who wish to add distributions to this guide, installation instructions are included here only for distributions with a reasonably up-to-date version of ``ansible``. The distribution MUST ensure that ``ansible-core`` and ``ansible`` versions are kept in sync to the extent that the distribution build system allows. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ Matrix room.
+To add instructions for another distribution to this guide, maintainers for that distribution **must** do the following:
+
+    * Ensure the distribution provides a reasonably up-to-date version of ``ansible``.
```

Comment:
I'm not sure if `up-to-date` is supposed to be hyphenated. @oraNod, @samccann, WDYT?

Reply:
had to google this one.. it's correct with the hyphens in this caste as it's acting as an adjective.